doc/man3/BIO_s_socket.pod

```diff
@@ -35,11 +35,12 @@ platforms sockets are not file descriptors and use distinct I/O routines,
 Windows is one such platform. Any code mixing the two will not work on
 all platforms.
 
-=head1 CTRLs
+=head1 CTRLS
```

Comment:
Maybe CTRL COMMANDS instead?

Reply:
Perhaps, I did however change them all to CTRLS before I say your reply.